.github/workflows/test_testflight.yml

```diff
@@ -0,0 +1,80 @@
+name: Test Upload Build to TestFlight
+
+# SECRETS needed:
+### SSH_PRIVATE_KEY for Match Repo
+### MATCH_REPO
+### MATCH_PASS
+### API_KEY_ID
+### ISSUER_ID
+### APPSTORE_CONNECT_API_KEY
+
+on:
+  push:
+    branches: [ feature/**, bug/**, chore/** ]
```

Comment:
I would prefer to test this on the `develop` branch only, otherwise, we will spam a lot of builds on the TestFlight page. What do you think?

Reply:
@suho Should we use on:pull_request instead? The idea is to run this test to check if the code submitted for PR can still build successfully, if it run on develop then the failed code might already be merge to develop.

Reply:
Yes, just think about this again, we could do with `on:pull_request` :D